docs/building-spark.md

```diff
@@ -144,6 +144,17 @@ The ScalaTest plugin also supports running only a specific test suite as follows
 
     mvn -Dhadoop.version=... -DwildcardSuites=org.apache.spark.repl.ReplSuite test
 
+# Building module
```

Comment:
"Building a single module"? and maybe say "submodules individually" to emphasize this isn't about building them all, which is what happens normally. For tidiness I'd back-tick `mvn -pl` and `spark-streaming_2.10`, `artifactId`, `streaming/pom.xml`. LGTM otherwise.